tests/test_mssqlcliclient.py

```diff
@@ -14,11 +14,19 @@
 )
 
 
-# All tests apart from test_mssqlcliclient_request_response require a live connection to an
-# AdventureWorks2014 database with a hardcoded test server.
-# Make modifications to mssqlutils.create_mssql_cli_client() to use a different server and database.
 # Please Note: These tests cannot be run offline.
-class MssqlCliClientTests(unittest.TestCase):
+
+class MssqlCliClient:   # pylint: disable=too-few-public-methods
```

Comment:
This class becomes inherited by the below test classes to reduce redundant code of mssqlcli client creation and teardown. It leverages "fixtures" in pytest which can be used as inputs for tests.